deploy/ansible/playbooks/roles/stellar-core/tasks/main.yml

```diff
@@ -48,3 +48,8 @@
 - name: start stellar-core
   docker_service:
     project_src: /opt/stellar-core
+
+- name: configure core parameters
+  uri:
+    url: http://localhost:11626/upgrades?mode=set&upgradetime=1970-01-01T00:00:00Z&basereserve=0&maxtxsize=500&protocolversion=9'
```

Comment:
maybe decouple this as three different tasks, and specify in the title what this is doing
also after each command, do somethign similar to what invoke is doing here [here](https://github.com/kinecosystem/blockchain-ops/blob/98735ed6297cbc14fefa2a15f116e4c6ec5d38b9/tasks.py#L230) continuously polling core for its latest ledger, and moving on only after the parameter has been applied

Reply:
but we're not guaranteed that the core will even start after the command. if you're setting a new core, it might have to wait for other nodes. so waiting is not always a good idea.

Reply:
alrighty then :+1:

Reply:
can you at least separate this into different upgrade tasks + add a TODO comment on the above?